internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -10,7 +10,6 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64default"
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema/setdefault"
```

Comment:
Changes to this file are due to TPF strictness and using the `/tenantUpgrade` endpoint, which would raise errors:

```
  | Error: Provider produced inconsistent result after apply
  | 
  | When applying changes to mongodbatlas_advanced_cluster.test, provider
  | "provider[\"registry.terraform.io/hashicorp/mongodbatlas\"]" produced an
  | unexpected new value: .create_date: was
  | cty.StringVal("2024-12-03T16:54:26Z"), but now
  | cty.StringVal("2024-12-03T16:57:44Z").
  | 
  | This is a bug in the provider, which should be reported in the provider's own
  | issue tracker.
  | 
  | Error: Provider produced inconsistent result after apply
  | 
  | When applying changes to mongodbatlas_advanced_cluster.test, provider
  | "provider[\"registry.terraform.io/hashicorp/mongodbatlas\"]" produced an
  | unexpected new value: .replication_specs[0].zone_name: was
  | cty.StringVal("ZoneName managed by Terraform"), but now cty.StringVal("Zone
  | 1").
  | 
  | This is a bug in the provider, which should be reported in the provider's own
  | issue tracker.
  | 
  | Error: Provider produced inconsistent result after apply
  | 
  | When applying changes to mongodbatlas_advanced_cluster.test, provider
  | "provider[\"registry.terraform.io/hashicorp/mongodbatlas\"]" produced an
  | unexpected new value: .cluster_id: was
  | cty.StringVal("674f37c288864967c85f6b47"), but now
  | cty.StringVal("674f388843ca6e2ab9803e84").
  ```